core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/InvokerHealthManager.scala

```diff
@@ -269,6 +270,7 @@ object InvokerHealthManager {
   val healthActionNamePrefix = "invokerHealthTestAction"
   val bufferSize = 10
   val bufferErrorTolerance = 3
+  var useMemory = 0l
```

Comment:
nit: `usedMemory`

Reply:
Ok, good catch, as i said above, i removed /memory api in this pr. i will add `/memory api` in another pr: `[New Scheduler] FunctionPullingContainerPool` with `functionPullingContainerPool.ask(UsedMemory)(Timeout(5.seconds)).mapTo[Int]`.